source/sio/net/ip/address.hpp

```diff
@@ -89,61 +95,138 @@ namespace sio::net::ip {
   class address_v6 {
    public:
     using bytes_type = std::array<std::byte, 16>;
+    using in6_addr_type = ::in6_addr;
 
     /// Obtain an address object that represents any address.
     static address_v6 any() noexcept {
       return address_v6();
     }
 
     /// Obtain an address object that represents the loopback address.
-    static address_v6 loopback() noexcept;
+    static address_v6 loopback() noexcept {
+      address_v6 __any{};
```

Comment:
I copied this file originally from a private branch of stdexec where identifiers needed to be uglified to target a standard library implementation. Here we don't need to prefix with `__` and it is even forbidden by the standard. So lets rename all `__foo` to `foo` and similarily all `_Bar` to `Bar`

Reply:
yes, thanks.